### DISCUSSION TITLE
Firefox (Android version) on ChromeOS: settings not changable

### DISCUSSION THREAD

**Discussion by wehkah** [uBlockOrigin/uAssets#15473] [Q&A] [answered]
Hello,

does any one else use FF on a Chromebook? On native Android devices, including my phone, I can use the settings of uBlock the same way as I can use them on my PC. But on my Chromebook the settings can't be changed:

- they look different: there's just one column on the right showing "+" or "-", while on other devices there are two columns, one for "this site only", one for "all sites", and both cells in each row can be changed from green to yellow and to red

- no matter where I click, these settings in this one column won't change

Has anyone managed to change the settings there? What am I doing wrong there? Thanks in advance for any clue and help!

**Comment by ghost** [comment, marked as answer]
https://github.com/uBlockOrigin/uBlock-issues/wiki/Quick-guide:-popup-user-interface#the-overview-panel

You are not an advanced user.

**Comment by wehkah** [comment]
Oh, dear! How could I miss that? Must be due to low caffeine level!

Thanks for the hint!
